src/rpc/blockchain.cpp

```diff
@@ -2316,7 +2331,7 @@ UniValue dumptxoutset(const JSONRPCRequest& request)
 
         ::ChainstateActive().ForceFlushStateToDisk();
 
-        if (!GetUTXOStats(&::ChainstateActive().CoinsDB(), stats, RpcInterruptionPoint)) {
+        if (!GetUTXOStats(&::ChainstateActive().CoinsDB(), stats, CoinStatsHashType::HASH_SERIALIZED, RpcInterruptionPoint)) {
```

Comment:
in commit ea41a5bdcc49566be0c23ef9362781e70f557bad:

why is this not changed to NONE?

Reply:
Agreed, this hash is unused in UTXO snapshots, as judged by the code as well as the hardcoded UTXO set has in `rpc_dumptxoutset.py`.

Reply:
done